### PR TITLE
Update tuya.ts

### PR DESCRIPTION
**Pull request by RafaelDuncan**
added _TZE204_5cuocqty

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -7397,6 +7397,31 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+    fingerprint: [
+        {   modelID: 'TS0601',
+            manufacturerName: '_TZE204_5cuocqty',
```
Comment (Koenkk) [inline src/devices/tuya.ts:7403]: This can be merged with the already existing `TS0601_dimmer_5`

Comment (RafaelDuncan) [reply src/devices/tuya.ts:7403]: thank you @Koenkk for your fast reaction! 
Does this mean that I don't have to do anything for now?

Comment (RafaelDuncan) [reply src/devices/tuya.ts:7403]: thanks @Koenkk . i just merged that change to the existing TS0601_dimmer_5.
I hope I did it right now...:)

Comment (Koenkk) [reply src/devices/tuya.ts:7403]: Looks good, will merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/7072

**Pull request closed without merging**